src/components/searchForm/searchForm.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { hot } from "react-hot-loader";
+import Button from "../shared/button/button";
+import TextField from "../shared/input/input";
+import RadioGroup from "../shared/radioGroup/radioGroup";
+import "./searchForm.scss";
+
+
+class SearchForm extends React.Component {
+    constructor (props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleSearchClick () {
+        console.log("Start search");
+    }
+
+    render () {
+        return (
+            <div className="searchForm">
+                <h2 className="searchForm__title">Find your movie</h2>
+                <TextField/>
+                <div className="searchForm__controls">
+                    <RadioGroup title="Search by"
+                        name="searchByField"
+                        radios={[
```

Comment:
Maybe move them to consts as these values are not gonna be changed?

Reply:
Done